plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoColumnHandle.java

```diff
@@ -29,28 +33,51 @@
 public class MongoColumnHandle
         implements ColumnHandle
 {
-    private final String name;
+    private final String baseName;
```

Comment:
nit: Changing `name` to `baseName` in a separate commit would be easier to review. It can be squashed before the merge if you like